README.md

```diff
@@ -33,7 +39,20 @@ then to type out this with:
 
 Yet, the two are equivalent using Contra.
```

Comment:
It's not quite homoiconicity, but the circularity is important.  Describe how this is useful (I know it is, I just can't come up with that right now.